netbeans/src/main/java/org/alice/netbeans/project/ProjectCodeGenerator.java

```diff
@@ -131,7 +129,7 @@ public static Collection<FileObject> generateCode(File aliceProjectFile, File ja
       }
     }
 
-    if (progressHandle != null) {
+      if (progressHandle != null) {
```

Comment:
Probably don't want this extra whitespace

Reply:
Oops. Thank you.